src/Kafka/ManageKafkaPermissions/components/ResourceTypeOperationFilter/ResourceTypeOperationFilter.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { useState } from "react";
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+
+import { ResourceTypeOperationFilter } from "./ResourceTypeOperationFilter";
+
+export default {
+  component: ResourceTypeOperationFilter,
+  args: {},
+} as ComponentMeta<typeof ResourceTypeOperationFilter>;
+
+const Template: ComponentStory<typeof ResourceTypeOperationFilter> = () => {
+  const [checkedItems, setCheckedItems] = useState([]);
+  return (
+    <ResourceTypeOperationFilter
+      setCheckedItems={setCheckedItems}
+      checkedItems={checkedItems}
+    />
+  );
+};
+
+export const ResourceTypeOperationFilterStory = Template.bind({});
+ResourceTypeOperationFilterStory.storyName = "ResourceTypeOperationFilter";
```

Comment:
Now that you can pass the checkedItems, it would be nice to have a few stories where we show how the menu look like depending on the various selection options.

Also, test the stories with jest tests. Check [here](https://github.com/redhat-developer/app-services-ui-components/blob/main/src/shared/FormatDate/FormatDate.test.tsx) for an example about how to do that.